pkgs/development/php-packages/uv/default.nix

```diff
@@ -0,0 +1,17 @@
+{ buildPecl, lib, libuv }:
+
+buildPecl {
```

Comment:
I have nothing against not specifying the `src` attribute since the `buildPecl` wrapper will automatically download the sources from `pecl.php.net`... but I would have prefered to use Github instead.

Reply:
Ok, it's fetched from GitHub now.